### PR TITLE
Soft assert added

### PR DESCRIPTION
**Pull request by pashidlos**
https://github.com/Visual-Regression-Tracker/Visual-Regression-Tracker/issues/63

### CODE REVIEW

File: tests/test_visualRegressionTracker.py [hunk 1]
```diff
@@ -27,6 +29,14 @@ def vrt():
     yield VisualRegressionTracker(CONFIG)
 
 
+@pytest.fixture
```
Comment (dcrowe) [inline tests/test_visualRegressionTracker.py]: Not big deal, but pytest already provides a fixture for logging. See: https://docs.pytest.org/en/stable/logging.html#caplog-fixture

File: visual_regression_tracker/visualRegressionTracker.py [hunk 2]
```diff
@@ -84,10 +87,17 @@ def _submitTestResult(self, test: TestRun) -> TestRunResult:
...
-            raise Exception(f'Difference found: {result.url}')
+        switcher = {
+            TestRunStatus.NEW: f'No baseline: {result.url}',
+            TestRunStatus.UNRESOLVED: f'Difference found: {result.url}'
+        }
+        error_message = switcher.get(result.status, '')
+
+        if error_message:
+            if self.config.enableSoftAssert:
+                logging.error(error_message)
```
Comment (dcrowe) [inline visual_regression_tracker/visualRegressionTracker.py]: The python docs recommend that libraries log to a named logger: https://docs.python.org/3/howto/logging.html#configuring-logging-for-a-library

**Comment by dcrowe** [comment]
I've made a commit to address my recommendations. What do you think?

File: tests/test_visualRegressionTracker.py (hunk 1 above)
Comment (pashidlos) [reply tests/test_visualRegressionTracker.py]: good to know!
gonna use it in future

File: visual_regression_tracker/visualRegressionTracker.py (hunk 2 above)
Comment (pashidlos) [reply visual_regression_tracker/visualRegressionTracker.py]: good catch!

**Comment by pashidlos** [comment]
thank you for help!

**Pull request merged**